api/src/pcapi/routes/backoffice_v3/forms/fields.py

```diff
@@ -168,12 +168,13 @@ class PCSelectMultipleField(wtforms.SelectMultipleField):
     validators = [validators.Optional()]
 
 
-class PCAutocompleteSelectMultipleField(PCSelectMultipleField):
-    widget = partial(widget, template="components/forms/select_multiple_field_autocomplete.html")
+class PCTomSelectField(PCSelectMultipleField):
+    widget = partial(widget, template="components/forms/tom_select_field.html")
 
-    def __init__(self, label: str, endpoint: str, **kwargs: typing.Any):
+    def __init__(self, label: str, endpoint: str, multiple: bool, **kwargs: typing.Any):
```

Comment:
```suggestion
    def __init__(self, label: str, endpoint: str, multiple: bool = False, **kwargs: typing.Any):
```
comme tu passes en paramètre nommé `multiple=True` chaque fois que nécessaire, autant laisser une valeur par défaut